aws/variables.tf

```diff
@@ -1,7 +1,3 @@
-variable "aws_region" {
```

Comment:
this is not just a syntax chage

Reply:
Yep, this var is unused

Reply:
but if you really like it, I can add it back